addons/escoria-core/game/core-scripts/esc_controller.gd

```diff
@@ -41,8 +41,8 @@ func perform_walk(
 				target_position = destination.node.interact_position
 				
 			var walk_context = ESCWalkContext.new(
-				destination.node, 
-				Vector2(), 
+				destination, 
+				target_position, 
```

Comment:
For some reason, this was initialized to Vector2() while we did have a target_position :/

Reply:
🤷